lib/stdio.js

```diff
@@ -1,5 +1,26 @@
+import {isStream} from 'is-stream';
+
 const aliases = ['stdin', 'stdout', 'stderr'];
 
+const isIterableStdin = stdinOption => typeof stdinOption === 'object'
+	&& stdinOption !== null
+	&& !isStream(stdinOption)
+	&& (typeof stdinOption[Symbol.asyncIterator] === 'function' || typeof stdinOption[Symbol.iterator] === 'function');
```

Comment:
We explicitly exclude `string` and `Stream` since those already have a meaning with the `stdin` option.